src/redux/store.js

```diff
@@ -1,32 +1,32 @@
 import { configureStore } from '@reduxjs/toolkit';
 import {
-  persistReducer,
-  FLUSH,
-  REHYDRATE,
-  PAUSE,
-  PERSIST,
-  PURGE,
-  REGISTER,
+    persistReducer,
+    FLUSH,
+    REHYDRATE,
+    PAUSE,
+    PERSIST,
+    PURGE,
+    REGISTER,
```

Comment:
why have unnecessary indentation changes ??? 
Are you using prettier configuration as configured?